app/views/tasks/index.html.erb

```diff
@@ -1,30 +1,22 @@
-<!-- NOTE TO SELF: This file can called .html.erb or .erb, html makes it clear that it'll end up as html.(?) -->
 <div class="task_list">
-<h1> Task List </h1>
-<table>
-  <% @all_tasks.each do |task| %>
-  <tr>
-    <td>
-      <%= link_to "#{task[:name]}", "/tasks/#{task[:id]}" %>
-      <% # NOTE TO SELF: 'link_to task[:name]' also works %>
-    </td>
-    <td>
-      <%= task[:completed_at].nil? ? @uncompleted : @completed %>
-    </td>
-    <td>
-      <%= button_to "DELETE", "/tasks/delete/#{task[:id]}", method: :get %>
-      <% # 'method: "get"' also works!  %>
-      <% # NOTE BELOW: Pops up a confirmation on the same page, chose not to use it though (commented it out). %>
-      <!-- <%= button_to "DELETE", "/tasks/#{task[:id]}", method: "delete", data: {:confirm => "ARE YOUS SUREEEE?"} %> -->
-    </td>
-  </tr>
-  <% end %>
-</table>
 
-<%= button_to "ADD", "/tasks/new", {method: :get, :class => "confirm"} %>
+  <ul>
+    <% @all_tasks.each do |task| %>
+      <li>
+        <%= link_to "#{task[:name]}", "/tasks/#{task[:id]}" %>
+        <% # NOTE TO SELF: 'link_to task[:name]' also works %>
+        <!-- <span class="<%= task[:completed_at].nil? ? "invisible" : "visible" %>">
+          <%= link_to "\u2714", "/tasks/#{task[:id]}", {method: :patch, class: "completed"}%>
+        </span> -->
+        <%= "-- #{task.person.name}" %>
+        <%= task[:completed_at].nil? ?
```

Comment:
I think you could use this ternary to set a local variable within the view to contain the string `"completed"` or `"not_completed". Then you would only need to write the `link_to` one time with the string variable.